{"name": "Проверка ошибки получения информации по заказу: запрос с несуществующим номером заказа", "status": "passed", "start": 1699380933740, "stop": 1699380933947, "uuid": "3f7c3844-fe30-4c72-bace-6515de1ef925", "historyId": "2c2535f50a1f0163a11d75abdf84f4b9", "testCaseId": "2c2535f50a1f0163a11d75abdf84f4b9", "fullName": "tests.test_get_one_order.TestGetOneOrder#test_get_one_order_incorrect_id_error_resp", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_one_order"}, {"name": "subSuite", "value": "TestGetOneOrder"}, {"name": "host", "value": "WKS-084"}, {"name": "thread", "value": "16844-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_one_order"}]}